styles/styles.css

```diff
@@ -0,0 +1,118 @@
+
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+h3 { 
+  display: block;
+  font-size: 1.10em;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  margin-left: 0;
+  margin-right: 0;
+  font-weight: bold;
+}
+
+.header {
+  align-items: center;
+  background-color: #FFF;
+  border-top: 3px solid #F5C85F;
+  display: flex;
+  justify-content: center;
+  padding: 20px 0 40px;
+  width: 100%;
+}
+
+.vertical-separator {
+  background: linear-gradient(#FFF, #F5C85F, #FFF);
+  width: 3px;
+  height: 300px;
+  margin: 0px 30px;
+} 
+
+.gray-divider {
```

Comment:
Instead of a whole class, this should be a pseudo-element since it's just for decoration purposes